weemoji.py

```diff
@@ -51,7 +51,7 @@ def replace_emoji(match):
     codepoint = EMOJI.get(text[1:-1])
     if codepoint:
         raw = '\\U%08x' % int(codepoint, 16)
-        return raw.decode('unicode-escape').encode('utf-8')
+        return raw.encode('utf-8').decode('unicode-escape')
```

Comment:
What does changing the ordering address?

Reply:
because Python 3 doesn't have `decode` as an instance of the string and require a decode first